scripts/Dockerfile

```diff
@@ -22,6 +22,7 @@ RUN sudo apt-get update && sudo apt-get install -yy \
 RUN wget "https://github.com/Shippable/support/raw/master/bin/reports" && \
     chmod +x reports && \
     rm -f /home/shippable/cexec/bin/reports && \
+	mkdir -p /home/shippable/cexec/bin && \
```

Comment:
@muvarov: this dir should already exist in the base image for the Dockerfile. Are you seeing the need to create the directory in your testing?